### PR TITLE
Fix issues with new weather card

### PR DESCRIPTION
**Pull request by c727**
fix: #1144 

- card is now more responsive
- forecast reduced to 5 days/hours
- replaced visibility with air pressure in main card
- minor design changes

### CODE REVIEW

**Comment by balloob** [comment]
Does this needs to be part of the beta?

File: src/cards/ha-weather-card.html [hunk 1]
```diff
@@ -217,4 +238,4 @@
   }
 }
 customElements.define(HaWeatherCard.is, HaWeatherCard);
-</script>
+</script>
```
Comment (balloob) [inline src/cards/ha-weather-card.html]: No new line at end of file.

**Review by balloob — APPROVED** [review]

**Comment by c727** [comment]
yes, please add this to the next release

**Comment by c727** [comment]
maybe, if we have 3-digit temperature and a language with long attribute names the card is too small :(

**Comment by arsaboo** [comment]
The bottom half of our card is perfect, but the top half is a little crowded. 
image

We can reduce the font weights for the temperature and the weather description and add some padding around the weather icon to make it less busy. Also, using mdi icons for all three strings, `Air Pressure`, `Humidity`, and `Wind Speed` will reduce some clutter. Just as an example, here's the card from [this thread](https://community.home-assistant.io/t/custom-ui-weather-state-card-with-a-question/23008) (ignore the animated icons)

image

**Comment by balloob** [comment]
Open a new issue 😉

**Pull request merged**